src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -102,16 +103,9 @@ public Optional<Pose2d> estimatePose(Rotation2d robotAngle) {
         return Optional.empty();
     }
 
-    /**
-     * updates the limelight log inputs periodically
-     */
-    public void periodic() {
-        updateInputs();
-    }
-
     @Override
     public String getSubsystemName() {
-        return "Limelight";
```

Comment:
Why not call this `Limelight` really? Other vision processing solutions may be used.